Changes

```diff
@@ -24,6 +24,8 @@ be mentioned in the 4.06 section below instead of here.)
 
 ### Manual and documentation:
 
+- PR#7647: emphasize ocaml.org website and forum in README
```

Comment:
If you want a Change entry, you should have a proper credit line like other entries.

Reply:
Ah, OK, done. I figured since this will be a user-facing change (?), it should have an entry.

Reply:
Yes, I agree that this deserves a Change entry!